atlas-sdk/src/main/java/com/atlas/sdk/view/AtlasViewFragment.kt

```diff
@@ -0,0 +1,129 @@
+package com.atlas.sdk.view
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Lifecycle
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.atlas.sdk.AtlasSdk
+import com.atlas.sdk.data.AtlasMessageHandler
+import com.atlas.sdk.data.AtlasUser
+import com.atlas.sdk.databinding.FragmentAtlasViewBinding
+
+class AtlasViewFragment : Fragment() {
+
+    internal var atlasSdk: AtlasSdk? = null
+    internal var appId: String? = null
+    internal var user: AtlasUser? = null
+
+    private var _binding: FragmentAtlasViewBinding? = null
+
+    private val binding get() = _binding!!
+
+    private var receiver: BroadcastReceiver? = null
+    private var intentFilter = IntentFilter().apply {
+        addAction(AtlasSdk.ON_CHANGE_IDENTITY_ACTION)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentAtlasViewBinding.inflate(inflater, container, false)
+        val root: View = binding.root
+
+        return root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.atlasView.setSdkAtlasMessageHandler(atlasSdk?.internalAtlasMessageHandler)
+        binding.atlasView.bindToLifeCycle(lifecycle)
+        appId?.let { binding.atlasView.applyConfig(it, user) }
+        binding.atlasView.openPage()
+
+        binding.atlasView.setAtlasMessageHandler(
+            object : AtlasMessageHandler() {
```

Comment:
Looks like this event handlers are not connected to any handlers that could be passed from outside to SDK

Reply:
I am not sure if that was implemented previously, but the expectation is that you can add them on AtlasSDK